app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        person_inst = Person.people[person["name"]]
```

Comment:
you make variable name shorter for few characters, suggest you just use full name:
```suggestion
        person_instance = Person.people[person["name"]]
```

Reply:
Done